Tilt dev setup fails during `make kind-create tilt-up`

/kind bug

**What steps did you take and what happened:**
I was following the developer documentation:
https://capz.sigs.k8s.io/developers/development

Specifically the section:
[Tilt for dev in CAPZ](https://capz.sigs.k8s.io/developers/development#tilt-for-dev-in-capz)

I ran the documented command:

```bash
make kind-create tilt-up
```

During the setup, the process fails with the following error:

```bash
./scripts/kind-with-registry.sh: line 159: ./scripts/../hack/tools/bin/azwi: No such file or directory
make: *** [kind-create] Error 1
```

**What did you expect to happen:**
The `make kind-create tilt-up` command should install or bootstrap all required tooling (including azwi), or clearly document any manual prerequisites required before running the command.

**Anything else you would like to add:**
This was encountered on a fresh environment while following the docs as a new contributor.
It’s not clear from the documentation whether `azwi` must be installed manually or why the binary is missing.
Happy to help fix the docs or tooling if pointed in the right direction.


**Environment:**

- cluster-api-provider-azure version: main branch (local clone)
```bash
❯ git log --oneline
1bd176862 (HEAD -> main, origin/main, origin/HEAD) Merge pull request #6018 from kubernetes-sigs/dependabot/go_modules/github.com/onsi/ginkgo/v2-2.27.3
```
- Kubernetes version: (use `kubectl version`):  NA (fails before cluster creation)
- OS (e.g. from `/etc/os-release`): macOS 26.2 (25C56)

--- a/Makefile
+++ b/Makefile
@@ -796,7 +796,7 @@ verify-container-images: ## Verify container images
 ##@ Tilt / Kind:
 
 .PHONY: kind-create
-kind-create: $(KUBECTL) ## Create capz kind cluster if needed.
+kind-create: $(KUBECTL) $(AZWI) ## Create capz kind cluster if needed.
 	./scripts/kind-with-registry.sh
 
 .PHONY: aks-create

